### PR TITLE
BJEJ-1 bunch of animations and modifications

### PR DESCRIPTION
**Pull request by LarryMatte**

### CODE REVIEW

**Comment by christiangoudreau** [comment]
Screeenss! :D Ask Jason how to do animated gifs

File: src/main/java/com/arcbees/bourseje/client/realtime/ui/Footer.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.arcbees.bourseje.client.realtime.ui;
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/bourseje/client/realtime/ui/Footer.java]: missing license header

File: src/main/java/com/arcbees/bourseje/client/realtime/ui/Header.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package com.arcbees.bourseje.client.realtime.ui;
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/bourseje/client/realtime/ui/Header.java]: missing license header

**Comment by christiangoudreau** [comment]
Small nits then LGTM

**Comment by LarryMatte** [comment]
Using LICEcap for animation screen capture is not the best option:
thanksanimation
Click on the link below and look in the attachment files to see a screen capture done with quicktime
https://arcbees.atlassian.net/browse/BJEJ-1

**Comment by christiangoudreau** [comment]
That's nice though :D LGTM!

**Comment by jasonlemay** [comment]
We get the general idea of the animation :+1:

File: src/main/java/com/arcbees/bourseje/client/realtime/ui/Footer.ui.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+             xmlns:g='urn:import:com.google.gwt.user.client.ui' xmlns:svg="urn:import:org.vectomatic.dom.svg.ui">
+
+    <ui:with field="res" type="com.arcbees.bourseje.client.resources.Resources"/>
+
+    <g:HTMLPanel tag="footer">
+        <div class="{res.styles.wrapper_desktop} {res.styles.clearfix}">
+            <div class="{res.styles.logoJccq}">
+                <svg:SVGImage resource="{res.logoJccq}"/>
+            </div>
+                <div class="{res.styles.logoArcbees}">
```
Comment (jasonlemay) [inline src/main/java/com/arcbees/bourseje/client/realtime/ui/Footer.ui.xml]: Strange indentation for the following lines

File: src/main/java/com/arcbees/bourseje/client/realtime/ui/Header.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+package com.arcbees.bourseje.client.realtime.ui;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.user.client.ui.HTMLPanel;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+
+
```
Comment (meriouma) [inline src/main/java/com/arcbees/bourseje/client/realtime/ui/Header.java]: rm nl

File: src/main/java/com/arcbees/bourseje/client/realtime/ui/Header.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2014 ArcBees Inc.
```
Comment (meriouma) [inline src/main/java/com/arcbees/bourseje/client/realtime/ui/Header.java]: 2015?

**Pull request merged**